gustaf/create/volumes.py

```diff
@@ -0,0 +1,93 @@
+"""gustaf/create/volumes.py
+
+Routines to create volumes.
+"""
+
+import numpy as np
+import random
+
+from gustaf.volumes import Volumes
+
+
+def extrude_tri_to_tet(tri, thickness = 1., layers = 1, randomize = False):
```

Comment:
with the new `quad_to_tri`, we can name this `extrude_to_tet` and just log the additional operations we do.